pkg/shared/diff_test.go

```diff
@@ -0,0 +1,91 @@
+package shared
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+)
+
+func Test_DiffAsJson(t *testing.T) {
```

Comment:
Is this function still used? I don't see any reference of it being called.

Reply:
Great question @JameelB 
The method was added [here](https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/commit/c388577e4db0b1fb49dcffb61cbc3ec338501743)\
But I haven't spotted any uses of it

Reply:
Is this method safe to remove @chirino?